### PR TITLE
i18n(es): Updated `components.mdx` & `i18n.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `components.mdx` & `i18n.mdx`
* [c2b2a17](https://github.com/withastro/starlight/commit/c2b2a1793246e976b82dcee9f2efa695352b48b9)

### CODE REVIEW

File: docs/src/content/docs/es/guides/components.mdx [hunk 1]
```diff
@@ -59,7 +61,7 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 
 Puedes mostrar contenido en una caja que coincida con los estilos de Starlight utilizando el componente `<Card>`. Envuelve varias tarjetas en el componente `<CardGrid>` para mostrar las tarjetas una al lado de la otra cuando hay suficiente espacio.
 
-Un `<Card>` debe tener un `title` y opcionalmente puede incluir un atributo `icon` establecido con el nombre de [uno de los iconos integrados de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Icons.ts).
+Un `<Card>` requiere un `title` y opcionalmente puede incluir un atributo `icon` establecido con el nombre de [uno de los iconos integrados de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Icons.ts).
```
Comment (dreyfus92) [inline docs/src/content/docs/es/guides/components.mdx:64]:
```suggestion
El componente `<Card>` requiere un `title` y opcionalmente puede incluir un atributo `icon` establecido con el nombre de [uno de los iconos integrados de Starlight](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Icons.ts).
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencia @Waxer59, buen trabajo 😁

**Comment by Waxer59** [comment]
Gracias por la revisión 🤠

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 2]
```diff
@@ -107,27 +109,43 @@ Cuando uses una `root` locale, coloca las páginas para ese idioma directamente
...
 
 ## Traduce la UI de Starlight
 
-Algunos de los elementos de la UI de Starlight requieren etiquetas de texto para funcionar.
-Por ejemplo, la tabla de contenido de esta página tiene un encabezado "En esta página" en inglés.
-Nosotros apuntamos a enviar estas etiquetas en tantos idiomas como sea posible, pero actualmente solo tenemos soporte para inglés, alemán, japonés y español.
-
-Puedes proporcionar traducciones para los idiomas adicionales que admites, o anular nuestras etiquetas predeterminadas, a través de la colección de datos `i18n`.
+Starlight proporciona de forma predeterminada traducciones para los textos utilizados en la UI al inglés, alemán, japonés, portugués y español. También damos la bienvenida a [contribuciones para agregar más idiomas predeterminados](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
```
[the remark this answers is not in the document]
Comment (Waxer59) [reply docs/src/content/docs/es/guides/i18n.mdx:143]: I have already added the French, what names do you mean?

[the remark this answers is not in the document]
Comment (delucis) [reply docs/src/content/docs/es/guides/i18n.mdx]: I think @Yan-Thomas meant that in English we write with capital letters “English, German…” but in Spanish these are lowercase “inglés, alemán…”. But I’m assuming that’s the standard way to write languages in Spanish?

[the remark this answers is not in the document]
Comment (Waxer59) [reply docs/src/content/docs/es/guides/i18n.mdx]: ah yes, in Spanish, the standard way to write the names of the languages is using lower case letters

**Review by delucis — APPROVED** [review]
Thanks again for this PR @Waxer59 💖

**Pull request merged**